cli/attestation.go

```diff
@@ -70,32 +120,261 @@ func (cli *CLI) NewGetAttestationCmd() *cobra.Command {
+	cmd.Flags().BoolVar(&cfg.Policy.RequireIdBlock, "require_id_block", defaultRequireIdBlock, "Require that the VM was launch with an ID_BLOCK signed by a trusted id key or author key")
+	cmd.Flags().BoolVar(&cfg.Policy.PermitProvisionalFirmware, "permit_provisional_software", defaultPermitProvisionalSoftware, "Permit provisional firmware (i.e., committed values may be less than current values).")
+	cmd.Flags().StringVar(&platformInfo, "platform_info", "", "The maximum acceptable PLATFORM_INFO field bit-wise. May be empty or a 64-bit unsigned integer")
+	cmd.Flags().StringVar(&cfg.Policy.MinimumVersion, "minimum_version", "", "Minimum AMD-SP firmware API version (major.minor). Each number must be 8-bit non-negative.")
+	cmd.Flags().StringArrayVar(&trustedAuthorKeys, "trusted_author_keys", []string{}, "Paths to x.509 certificates of trusted author keys")
+	cmd.Flags().StringArrayVar(&trustedAuthorHashes, "trusted_author_key_hashes", []string{}, "Hex-encoded SHA-384 hash values of trusted author keys in AMD public key format")
+	cmd.Flags().StringArrayVar(&trustedIdKeys, "trusted_id_keys", []string{}, "Paths to x.509 certificates of trusted author keys")
+	cmd.Flags().StringArrayVar(&trustedIdKeyHashes, "trusted_id_key_hashes", []string{}, "Hex-encoded SHA-384 hash values of trusted identity keys in AMD public key format")
+	cmd.Flags().StringVar(&cfg.RootOfTrust.Product, "product", "", "The AMD product name for the chip that generated the attestation report.")
+	cmd.Flags().StringVar(&stepping, "stepping", "", "The machine stepping for the chip that generated the attestation report. Default unchecked.")
+	cmd.Flags().StringArrayVar(&cfg.RootOfTrust.CabundlePaths, "CA_bundles_paths", []string{}, "Paths to CA bundles for the AMD product. Must be in PEM format, ASK, then ARK certificates. If unset, uses embedded root certificates.")
+	cmd.Flags().StringArrayVar(&cfg.RootOfTrust.Cabundles, "CA_bundles", []string{}, "PEM format CA bundles for the AMD product. Combined with contents of cabundle_paths.")
+
+	if err := cmd.MarkFlagRequired("report_data"); err != nil {
+		log.Fatalf("Failed to mark flag as required: %s", err)
+	}
+	return cmd
 }
 
-func verifyAndValidateAttestation(attestation, reportData []byte) error {
+func verifyAndValidateAttestation(attestation []byte) error {
+	sopts, err := verify.RootOfTrustToOptions(cfg.RootOfTrust)
+	if err != nil {
+		return err
+	}
+	sopts.Product = cfg.Policy.Product
+	sopts.Getter = &trust.RetryHTTPSGetter{
+		Timeout:       timeout,
+		MaxRetryDelay: maxRetryDelay,
+		Getter:        &trust.SimpleHTTPSGetter{},
+	}
 	attestationPB, err := abi.ReportCertsToProto(attestation)
 	if err != nil {
 		return err
 	}
-	if err = verify.SnpAttestation(attestationPB, verify.DefaultOptions()); err != nil {
+	if err = verify.SnpAttestation(attestationPB, sopts); err != nil {
+		return err
+	}
+	opts, err := validate.PolicyToOptions(cfg.Policy)
```

Comment:
This check returns an error. Field `cfg.Policy.Policy` should be initialized or set to default value. The default value is: `0x0000000000030000`. 

The value for `cfg.Policy.Policy` should be initialized by the option `guest_policy` or with the above mentioned default value.